reports/generator/generator_test.go

```diff
@@ -0,0 +1,306 @@
+package generator
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/golang/mock/gomock"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/reports"
+	generatorMock "github.com/iryonetwork/wwm/reports/generator/mock"
+	storageMock "github.com/iryonetwork/wwm/reports/mock"
```

Comment:
could not import github.com/iryonetwork/wwm/reports/mock (cannot find package "github.com/iryonetwork/wwm/reports/mock" in any of:
	/tmp/tmp.8HNu2UpNiH/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/reports/mock (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/reports/mock (from $GOROOT)
	/tmp/tmp.8HNu2UpNiH/src/github.com/iryonetwork/wwm/reports/mock (from $GOPATH))